dotnet-authserver/src/TeacherIdentity.AuthServer/wwwroot/Styles/site.scss

```diff
@@ -87,3 +87,72 @@ $app-button-inverse-hover-background-colour: govuk-tint($app-button-inverse-fore
 .empty-hyphens {
   hyphenate-character: '';
 }
+
+.moj-filter-layout {
+  @include govuk-clearfix;
+}
+
+.moj-filter-layout__filter {
+  box-shadow: inset 0 0 0 1px govuk-colour("light-grey"); // Extends the inset border left full height of the filters on mobile
+
+  @include govuk-media-query($until: desktop) {
+    margin-bottom: 20px;
```

Comment:
I added this to allow some space below the filter on mobile. Everything else was copied over